pkg/proxyagent/agent/agent.go

```diff
@@ -31,6 +31,7 @@ import (
 	"open-cluster-management.io/cluster-proxy/pkg/config"
 	"open-cluster-management.io/cluster-proxy/pkg/proxyserver/operator/authentication/selfsigned"
 	"open-cluster-management.io/cluster-proxy/pkg/util"
+	clusterv1beta2sdk "open-cluster-management.io/sdk-go/pkg/apis/cluster/v1beta2"
```

Comment:
Can we align the naming style with this line: https://github.com/open-cluster-management-io/clusteradm/blob/393f050c99ad3614567d4ea5650377876dcf8543/pkg/cmd/get/clusterset/exec.go#L20

Reply:
Done